dbutil.h

```diff
@@ -91,4 +91,11 @@ char * expand_homedir_path(const char *inpath);
 
 void fsync_parent_dir(const char* fn);
 
+#if DROPBEAR_MSAN
+/* FD_ZERO seems to leave some memory uninitialized. clear it to avoid false positives */
```

Comment:
So it seems this is just to silence the tool...
In this case sorry and please revert this change.

Reply:
Reverting.